Testreglar/3.3.3/Nett/nett-3.3.3a.json

```diff
@@ -0,0 +1,213 @@
+			"label": "Skjemaelement:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Blir informasjonen du har fylt ut liggende i skjemaelementet?",
+			"ht": "<p>Hovedregelen er at informasjon som er fylt ut skal bli liggende i skjemaet, selv om det blir oppdaget inndatafeil. Unntaket er dersom informasjonen er knyttet til personvern eller sikkerhet (sensitiv informasjon). </p>\n<p><strong>Merk:</strong> Det er ok at feil inndata blir korrigert automatisk.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G83",
+				"G84",
+				"G85"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "I skjemaelement der inndatafeil blir oppdaget automatisk, forsvinner inndata fra skjemaelementet."
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Får du tekstlig forslag til retting av inndatafeil?",
+			"ht": "<p>Forslag til retting av inndatafeil kan for eksempel vises</p>\n<ul>\n<li>i et modalvindu</li>\n<li>i ledeteksten til skjemaelement</li>\n<li>et annet sted i skjemaet/på siden, for eksempel på toppen</li>\n<li>når du navigerer i skjemaet</li>\n</ul>\n<p>Forslag til retting av inndatafeil kan for eksempel</p>\n<ul>\n<li>stå i direkte tilknytning til skjemaelementet, for eksempel i ledeteksten </li>\n<li>stå et annet sted på nettsiden</li>\n<li>være lenket til på en annen nettside. Slike lenker skal stå i nærheten av inndataelementet der feilen har oppstått.</li>\n</ul>\n<p><strong>Merk:</strong> Du skal ikke vurdere kvaliteten på forslaget.</p>\n<p>Det er ikke nok å identifisere feilen utelukkende med for eksempel </p>\n<ul>\n<li>et symbol i ledeteksten</li>\n<li>ved å endre farge</li>\n<li>visuell plassering av feilmeldingen</li>\n<li>deaktivert knapp for å sende inn eller gå videre i skjemaet</li>\n<li>lignende ikke-tekstlige indikatorer</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [
+				"G84",
+				"G85"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Er forlaget til retting kodet som tekst?",
+			"ht": "<p>Dette kan du sjekke på flere måter:</p>\n<p>Alternativ 1:</p>\n<ul>\n<li>Prøv å markere teksten med mus eller tastatur. Dette indikerer at testen er kodet som tekst, og ikke er et bilde av tekst.</li>\n</ul>\n<p>Alternativ 2:</p>\n<ul>\n<li>Åpne kodeverktøyet i nettleseren.</li>\n<li>Bruk \"select\" funksjonen i verktøyet og marker teksten.</li>\n<li>Sjekk at teksten er kodet som tekst, ved å se om du finner igjen teksten i koden.</li>\n</ul>\n<p>Alternativ 3:</p>\n<ul>\n<li>Dersom skjemaelementet har attributtet \"required\" eller aria-required=\"true\", kommer det fram en melding i nettleseren. Denne meldingen er kodet som tekst.</li>\n</ul>",
```

Comment:
Alternativ 3 er vel egentlig ikke et alternativ for å sjekke dette men en måte å oppnå samsvar på?

Reply:
Endret til merknad.